app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -1030,6 +1030,21 @@ a.mention-group {
   }
 }
 
+.topic-post-visited-line {
+  border-bottom: 1px solid var(--danger-medium);
+  line-height: 0.1em;
+  text-align: center;
+  margin: 1rem 0px;
+
+  span {
```

Comment:
Could we have a class for this please? I expect people might want to customise it.

Reply:
People can target it with `.topic-post-visited-line span` if they want to. Also, I'll prefer if we add the class only when there is a need for it. We have `.topic-list-item-separator` with a `td span` element as well for the topic stream.

Reply:
I'm sorry but no, if they want to style it, we need a class, directly styling elements cause lots of issues later in theming. This is an element which might get restyle and a such as "the need for it".

Reply:
I've added a class for the `span.`